.gitignore

```diff
@@ -15,12 +15,21 @@ dependency-reduced-pom.xml
 
 # Integration tests
 src/test/resources/integration/driver/oracle/*.jar
-src/test/resources/integration/driver/oracle/*.zip
+# src/test/resources/integration/driver/oracle/*.zip
```

Comment:
if it's no longer required, remove the line

Reply:
ok